lnwallet/channel.go

```diff
@@ -2522,6 +2522,23 @@ func (lc *LightningChannel) ForceClose() (*ForceCloseSummary, error) {
 	}, nil
 }
 
+// CreateRespCloseProposal is a wrapper function that determines the initial
+// fee that will be used for a cooperative channel close before generating the
+// proposed channel close transaction. This is called by the responder in a
+// cooperative channel close workflow.
+func (lc *LightningChannel) CreateRespCloseProposal() ([]byte, uint64, error) {
```

Comment:
Why's this needed? In the case of a responder to a cooperative closure workflow, they'd still need to be aware of the proposed fee, as using the prior (latest) fee-per-kw may be inaccurate in the case that the initiator chose a diff fee than was set initially within the channel.